glide.yaml

```diff
@@ -2,121 +2,101 @@ package: github.com/openshift/sdn
 import:
 # kube first
 - package: k8s.io/api
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0
 - package: k8s.io/apiextensions-apiserver
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0
 - package: k8s.io/apimachinery
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0
 - package: k8s.io/apiserver
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0
 - package: k8s.io/client-go
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0
 - package: k8s.io/cli-runtime
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0
 - package: k8s.io/cloud-provider
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0
 - package: k8s.io/cluster-bootstrap
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0
 - package: k8s.io/code-generator
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0
 - package: k8s.io/component-base
-  version: kubernetes-1.14.0
+  version: kubernetes-1.16.0
 - package: k8s.io/csi-api
-  version: kubernetes-1.14.0
+  version: master # FIXME
```

Comment:
So since this still doesn't have any v1.16 tag it should be pinned to the current commit SHA of master, not to "master" itself.
Except it seems like "`make update-deps && make`" still passes even if you remove the `k8s.io/csi-api` entry entirely. So just kill that I guess.